aries-site/src/pages/components/skeleton-loading.mdx

```diff
@@ -0,0 +1,33 @@
+import { Notification } from 'grommet';
+import { Example } from '../../layouts';
+import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
+
+<Example
+  componentName="Skeleton"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js"
+  docs="https://v2.grommet.io/skeleton?theme=hpe#props"
+>
+  <DashBoardSkeletonExample />
+</Example>
+
+### When to use
+Skeletons are used when:
+- Loading elements are known ahead of time but are not immediately available.
```

Comment:
When is an example of when we don't know what kind of elements we'd be rendering? It feels like generally, we know what elements will be rendered but we are waiting on the data to populate those elements

Reply:
🤔 hmm maybe we can reword @KennyAtHPE